libkbfs/disk_block_cache.go

```diff
@@ -1102,6 +1125,30 @@ func (cache *DiskBlockCacheLocal) DoesSyncCacheHaveSpace(
 	return limiterStatus.SyncCacheByteStatus.UsedFrac <= .99
 }
 
+// AddHomeTLF adds a TLF marked as "home" so that the blocks from it are
```

Comment:
You can use the ".. implements the DiskBlockCache interface..." style of comment here and below, for functions that have comments in the interface definition.  Otherwise we have to remember to update the comment in two places if it ever changes.

Reply:
So that's the rationale for those comments. I always find them mildly annoying as they introduce an extra step, but I guess it makes sense since we'd forget to update them.